TobogganApp.ui/src/styles/components/_productCards.scss

```diff
@@ -0,0 +1,52 @@
+.product-card { 
+    width: 100%;
+    height: 100%;
+    position: relative;
+    display: flex;
+    align-items: flex-end;
+    text-align: center;
+    transition: 0.5s ease-out;
+  }
+  
+  .product-card:before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: block;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(to bottom, rgbe(0,176,155,0.5), rgbe(150,201,61,1));
+    z-index: 2;
+    transition: 0.5s all;
+    opacity: 0;
+  }
+  
+  .product-card:hover:before {
+    background: #F0E4D8;
+    opacity: 1;
+  }
+  
+  .product-card .hover-content {
```

Comment:
This is dope! I was wondering how you did that

Reply:
: )